### DISCUSSION TITLE
Welcome to Quadrilaterals Discussions!

### DISCUSSION THREAD

**Discussion by cluce47** [cluce47/Quadrilaterals#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the ICS4U classes. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, add a comment below starting with your name. Comment on my project, code, ask questions, suggest changes, etc.

**Comment by psing1** [comment]
The code is looking good Ms. Luce! - Puneet

**Reply by Uros-Petrovic** [reply]
I agree +1

**Reply by NathanOmana** [reply]
Very insightful comment. I also agree that this code is great. I like 🍕 !

**Reply by sidharthsajan** [reply]
> Very insightful comment. I also agree that this code is great. I like 🍕 !

interesting comment. Your perspective is very thought provoking.

**Reply by LarryWg** [reply]
> > Very insightful comment. I also agree that this code is great. I like 🍕 !
> 
> interesting comment. Your perspective is very thought provoking.

What a genius person. This is an amazing comment sidharth you are amazing!

**Reply by Ajanzz** [reply]
nice

**Reply by SyphxN** [reply]
> interesting comment. Your perspective is very thought provoking.
Nah, disagree, I think the perspective is incredibly biased and wrong, and further action should be taken against the creator of the comment

**Comment by Uros-Petrovic** [comment]
Wow!! Very useful code, and well implemented. - Uros

**Reply by psing1** [reply]
I agree!

**Reply by LucisUC100** [reply]
Same here, I will try my best

**Comment by gle33** [comment]
Very cool and concise!!!! - Grace

**Reply by ciennava** [reply]
I agree!

**Comment by NathanOmana** [comment]
This code is very well written! Good job Ms.Luce. 👍

**Comment by aalsa16** [comment]
hi

**Reply by 1sazeem1** [reply]
hi

**Comment by liujulie13** [comment]
Nice code.

**Comment by richelle11** [comment]
The code is very organized and easy to read, I like the exception handling (●'◡'●)👍

**Comment by vnguy1** [comment]
Great code!! - Victoria

**Comment by djoga24** [comment]
I like how this code is able to cover all quadrilaterals. The code is well-written! I do wish there was a method for calculating perimeter too

**Reply by cluce47** [reply]
Thanks Dev - I'll try to include that.

**Comment by cluce47** [comment]
BTW all - This is an example of INHERITANCE with classes that I will be covering at a later date. BUT, feel free to clone or download my code to learn from it, test it, add to it, improve it, etc.

**Comment by maule1** [comment]
Hi, how is everyone? 😃

**Comment by azhan3** [comment]
ඞ

**Reply by NathanOmana** [reply]
ඞ I like 🍣

**Comment by ShriyaPatel18** [comment]
This is cool!

**Comment by mahita18** [comment]
Great code! I liked your inclusion of various methods :)👩‍💻💻⚙️

**Comment by RZhu917** [comment]
Very elegant

**Reply by qydan** [reply]
indeed

**Comment by sidharthsajan** [comment]
Incredible Code :)

**Comment by ciennava** [comment]
This code looks great!!

**Comment by arjunbala25** [comment]
This code is looking very sophisticated and well written!!! Good Job Ms. Luce

**Comment by FatimaMoussa101** [comment]
🍕

**Comment by n4mle** [comment]
namle i love coding and i love this code!! LOLLLL :pp

**Comment by Ajanzz** [comment]
This code looks good!!!!!

**Comment by tvien1** [comment]
The code written is very helpful for its purpose and works perfectly!

**Comment by LucisUC100** [comment]
Great code

**Comment by SyphxN** [comment]
I think the code can use more variations of quadrilaterals like trapezoids and kites.

**Comment by shanzz1** [comment]
Code is looking good Ms. Luce!!!!

**Comment by Aishaa15** [comment]
This code looks really good!! - Aisha

**Comment by smacd1** [comment]
Hai @NathanOmana

**Comment by mfati2** [comment]
Great code!!! -Mahnoor
